nuxt/components/form/Form.vue

```diff
@@ -61,6 +66,26 @@ const props = withDefaults(defineProps<Props>(), {
   submitName: undefined,
 })
 
+const store = useMaevsiStore()
+const turnstileKey = ref('')
+const isSubmitButtonDisabled = ref(true)
+
+watch(turnstileKey, (newKey) => {
```

Comment:
@dargmuesli (note to myself) TODO: document script setup order https://github.com/maevsi/maevsi/issues/895

Reply:
I've aligned the script setup block to the common format as described now in #1151